.github/workflows/release.yml

```diff
@@ -25,10 +25,10 @@ jobs:
           fetch-depth: 0
       - name: Detect Version for Docker
         id: docker-version
-        run: echo ::set-output name=VERSION::$(SEP="-" scripts/version)
+        run: echo "VERSION=$(SEP="-" scripts/version)" >> $GITHUB_STATE
```

Comment:
Don't we want `$GITHUB_OUTPUT` for `set-output`?

Reply:
I based it off https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/